build_scripts/docker_image_creator.py

```diff
@@ -0,0 +1,65 @@
+"""
+Script to run docker image creation
+Run the command:
+    python docker_image_creator.py docker_file_github_link optimized_binary_link gpu|cpu tensorflow_version python_version
+"""
+import argparse
+import os
+
+def create_docker_image(dockerfile_github_link, optbin_link, processor, framework_version, python_version):
+    """
+    Function builds a docker image with the TF optimized binary
+
+    :param dockerfile_github_link: link to docker file repo
+    :param optbin_link: link to where the optimized binary is
+    :param processor: gpu or cpu
+    :param framework_version: tensorflow version i.e 1.6.0
+    :param python_version: version of python to build container with i.e. 3.6.5 or 2.7.4
+    :return: final built imageid
+    """
+    # 0.) Initialize some commonly used variables
+    dockerfile_repo_name = dockerfile_github_link.split('/')[-1].split('.')[0]
+    pyV = "py{}".format(python_version.split('.')[0]) # i.e. py2
+    framework = "tensorflow"
+    # 1.) Clone dockerfile repo from Github
+    print("Cloning dockerfile repo...")
+    os.system("git clone {}".format(dockerfile_github_link))
```

Comment:
We can assume that the repository is already cloned, before someone is running this script.

Reply:
Ok I can refactor this.

Reply:
Can we work on the assumption that the script is run inside the build scripts directory?

Reply:
Hmmm, I don't know if that assumption should hold valid. This script should assume that it is within the build_scripts directory, but not necessarily executed from this directory. You can either update the documentation explaining that it should only be executed from a specific directory or update the script to be independent of the execution path and reliant on the location of itself. Your choice.